### DISCUSSION TITLE
Query on fake news dataset

### DISCUSSION THREAD

**Discussion by payelsantra** [pyg-team/pytorch_geometric#4373] [Q&A] [answered]
Hello, I recently went through the work User Preference-aware Fake Detection(UPFD). I was going through that git hub repository: https://github.com/safe-graph/GNN-FakeNews.git. I was trying to crawl the historical tweets using tweepy. I took news as a root node and tried to find the user-ids. I got the 1-hop connection of users and also found the follower network of those users. But cannot understand how to detect that, which followers are actually propagating the particular news (basically I am trying to find 2/3-hop connection). How to solve this problem?

**Comment by rusty1s** [comment]
I sadly cannot give you any advice on this one. How about asking for help in the linked GitHub repo? Pinging @YingtongDou as well.

**Comment by payelsantra** [comment]
Thank you for your response.

**Comment by YingtongDou** [comment, marked as answer]
Answered in https://github.com/safe-graph/GNN-FakeNews/discussions/15#discussioncomment-2493585
